standard_contracts/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+
+name = "mirai-standard-contracts"
+version = "0.0.1"
+authors = ["Herman Venter <hermanv@fb.com>"]
+description = "Specifications of standard library functions for MIRAI"
+repository = "https://github.com/facebookexperimental/MIRAI"
+readme = "README.md"
+license = "MIT"
+edition = "2018"
```

Comment:
Should the edition be 2019 now?

Reply:
The edition refers to the version of Rust that must be used to compile this. 2018 still seems appropriate.